iteration/iteration_test.go

```diff
@@ -222,7 +218,7 @@ func (s *TestIterationRepository) TestListIterationBySpace() {
 	e := repo.Create(context.Background(), &iteration.Iteration{
 		Name:       "Other Spring #2",
 		SpaceID:    anotherSpaceCreated.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c